src/screens/TouristSpot/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { ScrollView, View } from 'react-native';
+
+import { Header } from '../../components/Header/Header';
+import { TitleImage } from '../../components/TitleImage';
+import { Content } from '../TouristSpot/components/Content';
+import { Border, Button, ViewContainer, Text, Label } from './styles';
+
+import Feather from '@expo/vector-icons/Feather';
+
+export function TouristSpot() {
+  return (
```

Comment:
Create a function that opens the maps of the person with the informed latitudes, use something like:
```ts
const scheme = Platform.select({ ios: 'maps:0,0?q=', android: 'geo:0,0?q=' });
const latLng = `${lat},${lng}`;
const label = 'Custom Label';
const url = Platform.select({
  ios: `${scheme}${label}@${latLng}`,
  android: `${scheme}${latLng}(${label})`
});

Linking.openURL(url); 
```
(code taken from StackOverflow for example purposes, remember to make changes accordingly)